test/lib/ufe/testAttributes.py

```diff
@@ -183,6 +183,64 @@ def testAddRemoveAttribute(self):
         with self.assertRaisesRegex(KeyError, "Attribute 'MyAttribute' does not exist") as cm:
             attr = ball35Attrs.attribute("MyAttribute")
 
+    @unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') < '4024', 'Test for UFE preview version 0.4.24 and later')
```

Comment:
Should really say something about why the test needs 0.4.24. Such as "Test requires xxx feature only available on Ufe 0.4.24 and later"

Reply:
I update it.